sw/simulator/nps/nps_sensors_utils.c

```diff
@@ -8,7 +8,7 @@ void UpdateSensorLatency(double time, gpointer cur_reading, GSList **history, do
   /* add new reading */
   struct BoozDatedSensor *cur_read = g_new(struct BoozDatedSensor, 1);
   cur_read->time = time;
-  cur_read->value = (struct DoubleVect3 *) g_memdup(cur_reading, sizeof(struct DoubleVect3));
+  cur_read->value = (struct DoubleVect3 *) g_memdup2(cur_reading, sizeof(struct DoubleVect3));
```

Comment:
this is not working for Semaphore build

Reply:
I tried that, this function does not exist in Ubuntu 20.04.